diff --git a/install.py b/install.py
--- install.py
+++ install.py
@@ -248,9 +248,9 @@
 				raise SystemExit(f"{TEXT_RED}{TEXT_CROSS} {TEXT_BOLD}{selected_theme}{TEXT_RESET}{TEXT_RED} theme not found.{TEXT_RESET}")
 			if not selected_theme_assets.exists():
 				raise SystemExit(f"{TEXT_RED}{TEXT_CROSS} {TEXT_BOLD}{selected_theme}{TEXT_RESET}{TEXT_RED} theme assets not found.{TEXT_RESET}")

-	with TemporaryDirectory() as tmpdir:
+	with (Path(os.environ['NIX_OUT']) / 'build') as tmpdir:
 		tmp = Path(tmpdir)
 		sourcedir = tmp / SKIN_DIR
 		print(f"{TEXT_BLUE}{TEXT_ARROW} Copying to the stage directory {TEXT_BOLD}{sourcedir}{TEXT_RESET}")
 		shutil.copytree(skindir, sourcedir)
@@ -285,17 +285,5 @@
 			replace_css_colors(sourcedir / CSS_FILE, config)
 			replace_vgui_colors(sourcedir / COLORS_FILE, config)
 			shutil.copytree(selected_theme_assets, sourcedir / ASSETS_DIR, dirs_exist_ok=True)

-		targets = set()
-
-		for t in args.target:
-			if t == "normal":
-				targets.add(Path(TARGET_NORMAL).expanduser().resolve())
-			elif t == "flatpak":
-				targets.add(Path(TARGET_FLATPAK).expanduser().resolve())
-			else:
-				targets.add(Path(t).expanduser().resolve())
-
-		for target in targets:
-			install(sourcedir, target, args.name)
 		print(f"{TEXT_GREEN}{TEXT_CHECK} Done!{TEXT_RESET}")
