diff --git a/install.py b/install.py
--- install.py
+++ install.py
@@ -287,9 +287,9 @@
 			selected_theme = colorthemedir / t / "{}{}".format(t, ".theme")
 			if not selected_theme.exists():
 				raise SystemExit(f"{TEXT_RED}{TEXT_CROSS} {TEXT_BOLD}{selected_theme}{TEXT_RESET}{TEXT_RED} theme not found.{TEXT_RESET}")

-	with TemporaryDirectory() as tmpdir:
+	with (Path(os.environ['NIX_OUT']) / 'build') as tmpdir:
 		tmp = Path(tmpdir)
 		sourcedir = tmp
 		print(f"{TEXT_BLUE}{TEXT_ARROW} Creating stage directory {TEXT_BOLD}{sourcedir}{TEXT_RESET}")

@@ -306,18 +306,5 @@
 			config = configparser.ConfigParser()
 			config.read(selected_theme)
 			replace_css_colors(sourcedir / LIBRARY_CSS_FILE, config)

-		targets = set()
-
-		for t in args.target:
-			if t == "normal":
-				targets.add(Path(TARGET_NORMAL).expanduser().resolve())
-			elif t == "flatpak":
-				targets.add(Path(TARGET_FLATPAK).expanduser().resolve())
-			else:
-				targets.add(Path(t).expanduser().resolve())
-
-		for target in targets:
-			patch_client_css(sourcedir, target, "Library")
-
 		print(f"{TEXT_GREEN}{TEXT_CHECK} Done!{TEXT_RESET}")
