diff --git a/src/video_core/texture_cache/image_info.cpp b/src/video_core/texture_cache/image_info.cpp
index e8ddde6..4353eae 100644
--- a/src/video_core/texture_cache/image_info.cpp
+++ b/src/video_core/texture_cache/image_info.cpp
@@ -115,6 +115,13 @@ ImageInfo::ImageInfo(const TICEntry& config) noexcept {
         rescaleable &= (block.depth == 0) && resources.levels == 1;
         rescaleable &= size.height > 256 || GetFormatType(format) != SurfaceType::ColorTexture;
         downscaleable = size.height > 512;
+
+        // Disable rescaling if this is a TotK 1080p AO texture (height = 1080 / 4)
+        // Try narrowing to just the AO texture by checking some characteristics of it:
+        if (size.width == 480 && size.height == 270 && size.depth == 1 &&
+            config.border_size.Value() == 7) {
+            rescaleable = false;
+        }
     }
 }

@@ -156,6 +163,13 @@ ImageInfo::ImageInfo(const Maxwell3D::Regs::RenderTargetConfig& ct,
         downscaleable = size.height > 512;
         type = ImageType::e2D;
         resources.layers = ct.depth;
+
+        // Disable rescaling if this is a TotK 1080p AO texture (height = 1080 / 4)
+        // Try narrowing to just the AO texture by checking some characteristics of it:
+        if (size.width == 480 && size.height == 270 && size.depth == 1 &&
+            ct.format == Tegra::RenderTargetFormat::R32_FLOAT && ct.volume.Value() == 0) {
+            rescaleable = false;
+        }
     }
 }

